Index: xbanish/Makefile
===================================================================
--- xbanish.orig/Makefile
+++ xbanish/Makefile
@@ -4,9 +4,9 @@ CC	?= cc
 CFLAGS	?= -O2
 CFLAGS	+= -Wall -Wunused -Wmissing-prototypes -Wstrict-prototypes -Wunused
 
-PREFIX	?= /usr/local
+PREFIX	?= /usr
 BINDIR	?= $(PREFIX)/bin
-MANDIR	?= $(PREFIX)/man/man1
+MANDIR	?= $(PREFIX)/share/man/man1
 
 INSTALL_PROGRAM ?= install -s
 INSTALL_DATA ?= install
